lib/MongoDB/_URI.pm

```diff
@@ -22,7 +22,6 @@ our $VERSION = 'v2.2.1';
 use Moo;
 use MongoDB::Error;
 use Encode ();
-use Time::HiRes qw(time);
```

Comment:
Rather than reduce our precision, how about we convert `expires` to `Num` instead of `Int`?

Reply:
Hmm yea thats a better plan, will do that instead and rebase